permissions/plugin-byteguard-build-actions.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "byteguard-build-actions"
+github: "jenkinsci/byteguard-build-actions-plugin"
+paths: 
+- "org/jenkins-ci/plugins/byteguard-build-actions"
```

Comment:
Mismatch to POM contents. Please review this repo's readme what is required here.